app/api/services/reports/suppliers.py

```diff
@@ -67,9 +84,35 @@ def get_suppliers(self):
                 Supplier.status,
                 Supplier.creation_time,
                 Supplier.data['seller_type']['sme'].astext.label('sme'),
-                subquery.columns.categories
+                Supplier.website,
+                Supplier.linkedin,
+                Address.address_line,
+                Address.suburb,
+                Address.state,
+                Address.postal_code,
+                Supplier.data['number_of_employees'],
+                Supplier.data['seller_type']['start_up'].astext.label('start_up'),
+                Supplier.data['seller_type']['nfp_social_enterprise'].astext.label('not_for_profit'),
+                Supplier.data['regional'],
+                Supplier.data['travel'],
+                Supplier.data['seller_type']['disability'].astext.label('disability'),
+                Supplier.data['seller_type']['female_owned'].astext.label('female_owned'),
+                Supplier.data['seller_type']['indigenous'].astext.label('indigenous'),
+                Supplier.data['representative'],
+                Supplier.data['email'],
+                Supplier.data['phone'],
+                Supplier.data['contact_name'],
+                Supplier.data['contact_email'],
+                Supplier.data['contact_phone'],
+                Product.name.label('product_name'),
+                Product.summary.label('product_summary'),
+                Product.website.label('product_website'),
+                Product.pricing.label('product_pricing_link'),
+                subquery.columns.categories,
             )
             .join(subquery, Supplier.id == subquery.columns.supplier_id)
```

Comment:
```suggestion
            .outerjoin(subquery, Supplier.id == subquery.columns.supplier_id)
```

this might need to be outer because not all suppliers have domains